drracket/scribblings/drracket/extending.scrbl

```diff
@@ -31,8 +31,25 @@ DrRacket supports two forms of extension to the programming
   creating @tech{plugin}s, see @other-manual['(lib
   "scribblings/tools/tools.scrbl")].}
 
+@item{The @tech{Scripts} menu provided by the Quickscript plugin, makes
+  it easy to extend DrRacket with small Racket scripts that can be used
+  in the definition (or interaction) window, or to graphically interact
+  with the user. For information on creating @tech{scripts}, see 
+  @other-doc['(lib "quickscript/scribblings/quickscript.scrbl")].}
+  
+@item{You can add user defined keybindings. For information on creating 
+  user defined keybindings, see  
+  @secref["defining-shortcuts" #:doc '(lib "scribblings/drracket/drracket.scrbl")].}
+  
 ]
 
+Don't forget you can also define new color schemes for DrRacket 
+@secref["color-scheme" #:doc '(lib "scribblings/drracket/drracket.scrbl")].
+Existing colorschemes packages are  
+@hyperlink["https://pkgd.racket-lang.org/pkgn/search?tags=colorscheme"]{@italic{tagged 
+'colorscheme'}}.
```

Comment:
I think you need to replace the first single quote with a backtick to get it rendered correctly.

Reply:
thanks